pootle/static/js/shared/components/FormSelectInput.js

```diff
@@ -41,19 +44,23 @@ const FormSelectInput = React.createClass({
   /* Handlers */
 
   handleChange(value) {
-    this.props.handleChange(this.props.name, value);
+    const newValue = this.props.multiple ? value.split(',') : value;
+    this.props.handleChange(this.props.name, newValue);
   },
 
 
   /* Layout */
 
   render() {
+    const { value } = this.props;
+    /* FIXME: react-select#25 prevents using non-string values */
+    const selectValue = this.props.multiple ? value : value.toString();
     return (
       <Select
         onChange={this.handleChange}
         {...this.props}
-        /* FIXME: react-select#25 prevents using non-string values */
-        value={this.props.value.toString()}
+        multi={this.props.multiple}
```

Comment:
why make it more difficult for others to understand the code?

https://github.com/JedWatson/react-select#multiselect-options

"""You can enable multi-value selection by setting multi={true}."""